build.gradle

```diff
@@ -102,6 +105,12 @@ allprojects {
             force "com.google.guava:guava:" + guavaVersion
             force "org.glassfish.jersey.core:jersey-common:" + jerseyVersion
             force "org.glassfish.jersey.core:jersey-server:" + jerseyVersion
+            force "org.apache.zookeeper:zookeeper:" + apacheZookeeperVersion
+            force "io.netty:netty-transport:"+ nettyVersion
```

Comment:
the reason for this change is because the transitive dependency for zookeeper is netty `4.1.29.Final` whereas pravega uses `4.1.30.Final`, right ?

Reply:
Correct. The same reason applies for `boringSSL`.

Reply:
https://github.com/apache/zookeeper/blob/release-3.5.5/pom.xml uses nettyVersion as `4.1.29.Final`. for `netty-all` artifact. 
And `netty-all` artifact is used by `shared:controller-api`  which currently uses `4.1.30.Final`. this implies we should have a force dependency on `netty-all`.

Reply:
Sorry, it was before but I replaced it by the `netty` one. I restored it.